tools/setup_env.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Script setups the environment for the Ziggurat test suite
+
```

Comment:
I have this habit of adding `set -e` at the beginning. Could be considered here.

Reply:
I am checking the return value on a few places, like on line 42:
```
# Verify the algod binary path using the version option
$ALGORAND_BIN_PATH/$ALGOD_BIN_NAME -v $> /dev/null
if [ "$?" != "0" ]; then
    echo "Aborting. Cannot find $ALGORAND_BIN_PATH/$ALGOD_BIN_NAME".
    exit 1
fi
```

That wouldn't be possible if the script died because algod wasn't found, right?

Reply:
Yep, doesn't work with `set -e`

Reply:
You could always do `set +e` before and `set -e` after, but it becomes a bit funny this way.

Reply:
Hmm, didn't know that's possible either. Let me see.

Yeah, I'm sold. I like it. I can leave a small explanation like this:
```
set +e # Need to check the return result below

# Verify the algod binary path using the version option
$ALGORAND_BIN_PATH/$ALGOD_BIN_NAME -v $> /dev/null

if [ "$?" != "0" ]; then
    echo "Aborting. Cannot find $ALGORAND_BIN_PATH/$ALGOD_BIN_NAME".
    exit 1
fi
set -e    # Restore 
```

Reply:
Okay, I find this solution more elegant:
```
# Verify the algod binary path using the version option
set +e; $ALGORAND_BIN_PATH/$ALGOD_BIN_NAME -v &> /dev/null; RET=$?; set -e;  # Check command result
if [ "$RET" != "0" ]; then
    echo "Aborting. Cannot find $ALGORAND_BIN_PATH/$ALGOD_BIN_NAME".
    exit 1
fi
```
What do you think?

Reply:
It's a matter of taste really. Looks funny, but if helps then why not?